src/twisted/web/http.py

```diff
@@ -1890,12 +1889,21 @@ class _ChunkedTransferDecoder:
         state transition this is truncated at the front so that index 0 is
         where the next state shall begin.
 
-    @ivar _start: While in the C{'CHUNK_LENGTH'} state, tracks the index into
-        the buffer at which search for CRLF should resume. Resuming the search
-        at this position avoids doing quadratic work if the chunk length line
-        arrives over many calls to C{dataReceived}.
+    @ivar _start: While in the C{'CHUNK_LENGTH'} and C{'TRAILER'} states,
+        tracks the index into the buffer at which search for CRLF should resume.
+        Resuming the search at this position avoids doing quadratic work if the
+        chunk length line arrives over many calls to C{dataReceived}. Not used
+        in any other state.
+
+    @ivar _trailerHeaders: Accumulates raw/unparsed trailer headers.
+        See https://github.com/twisted/twisted/issues/12014
+
+    @ivar _totalTrailerHeadersSize: Maximum bytes for trailer header from the
```

Comment:
Minor comment

Maybe call it "maximum" if this is about max limit :)

Otherwise, you might thing this is the current lenght.

```suggestion
    @ivar _maximumTrailerHeadersSize: Maximum bytes for trailer header from the
```